playbooks/group_vars/backup-pruner/public.yml

```diff
@@ -1,19 +1,2 @@
+COMMON_SERVER_NO_BACKUPS: true
 COMMON_SERVER_INSTALL_CONSUL: false
-
-TARSNAPPER_CONF: /etc/tarsnapper.yml
```

Comment:
Was this completely unused?

Reply:
An `rgrep` in the secrets repo yielded only this single mesh, so yes, it was completely unused, including by third-party roles.